frontend/src/state/models/schedule-data/schedule-error-message.model.ts

```diff
@@ -0,0 +1,4 @@
+export interface ScheduleErrorMessageModel {
```

Comment:
Z tego co rozumiem, to `ScheduleErrorModel` przychodzi z backendu i potem zostaje przemapowany na `ErrorMessageModel`, który potem jest używany na froncie. Jeżeli tak, to `ScheduleErrorMessageModel` ma za mało pół, żebym mógł jakoś to użyć z tabelą.

Bardzo przydały by się jeszcze te trzy 
![image](https://user-images.githubusercontent.com/21079319/93373968-5f6bd600-f856-11ea-866c-6f6d9a65c904.png)